docs/website/content/integrations/lula.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Lula
+subtitle: The Cloud-Native Compliance Engine
+software:
+- lula
+inventors:
+- defenseunicorns
+labels:
+  category: compliance
+tutorials: # optional, links to tutorials for the integration
+- https://lula.dev
```

Comment:
```suggestion
```

I'd be inclined to leave this as the getting started link alone to keep it focused and include a link to the main site in the description below (`Automated Gov...`)